src/Sylius/Bundle/AdminBundle/Resources/views/Product/Show/_header.html.twig

```diff
@@ -9,12 +9,36 @@
         </h1>
         {% include "@SyliusAdmin/Product/Show/_breadcrumb.html.twig" %}
     </div>
+    {% set protocol = 'http://' %}
+    {% set counter = 0 %}
+    {% for server in configuration.request.server %}
+        {% set counter = counter + 1 %}
+        {% if  counter == 5%}
+            {% if 'HTTPS' in server %}
+                {% set protocol = 'https://' %}
+            {% endif %}
+        {% endif %}
+    {% endfor %}
```

Comment:
cc @Zales0123 what are you thinking about this solve, its looks like cancer but i think that its working ??

Reply:
Why is it needed? Can we just skip protocol and leave the browser to assign one of them?